src/transactionBundle.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Defines the TransactionBundle class which will allow us to add resources to
+ * a txn bundle in the proper format and convert the representation to JSON
+ */
+class TransactionBundle {
+  constructor() {
+    this.entry = [];
+  }
+
+  addEntryFromResource(resource) {
+    const newEntry = {
+      resource: resource,
+      request: {
+        method: 'POST',
+        url: resource.resourceType
```

Comment:
I think we should probably avoid hardcoding the entry request method to `POST`. This is always the behavior for submit-data, but if this class gets re-used for any other operations, we might want to do the `PUT` request? We could add another argument for the method to the function body, and handle the following cases:

* If it's `POST`, keep what you have
* If it's `PUT`, the `url` should become `` `${resource.resourceType}/${resource.id}` `` (if `resource.id` doesn't exist, existing `PUT` logic should already handle that error?)